### PR TITLE
2278 lesmis homepage

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- add lesmis home page
- move header and footer into page routes so each view can have different header and footer

---

### Screenshots:
see issue

### CODE REVIEW

File: packages/lesmis/src/components/FavouritesMenu.js [hunk 1]
```diff
@@ -5,9 +5,10 @@
...
 import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
-import { Typography, IconButton, Drawer, List, ListItem } from '@material-ui/core';
-import { Close, Favorite } from '@material-ui/icons';
+import { Typography, IconButton, Drawer } from '@material-ui/core';
+import { Close, StarBorder } from '@material-ui/icons';
 import { FlexStart } from './Layout';
 
 /** =================================
```
Comment (tcaiger) [inline packages/lesmis/src/components/FavouritesMenu.js:14]: This is just a placeholder component and doesn't need review at this time. There is another user story which covers it.

Comment (edmofro) [reply packages/lesmis/src/components/FavouritesMenu.js:14]: Can we remove it for now? It's not part of MVP, so it's probably best for it not to be released with potentially confusing placeholder info

File: packages/lesmis/src/views/HomeView.js [hunk 2]
```diff
@@ -3,11 +3,134 @@
...
+import { SearchBar, FlexSpaceBetween, FlexStart } from '../components';
 
-export const HomeView = () => (
-  <Container>
-    <Typography variant="h1">Home View</Typography>
-  </Container>
-);
+const Wrapper = styled.div`
+  position: relative;
+  height: 800px; // fallback height for older browsers
```
Comment (edmofro) [inline packages/lesmis/src/views/HomeView.js:15]: Glad you knew this was required 🙌

**Review by edmofro — APPROVED** [review]
Code all LGTM, have asked some questions on the issue. Probably best to wait for them before merging, but approved.

**Pull request merged**